conference/tw/HITCON/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Hacks In Taiwan Conference",
+  "title": "HITCON",
```

Comment:
Let's revise name & title. Like https://github.com/TGmeetup/TGmeetup/blob/master/community/tw/DigitalOceanHsinchu/package.json#L2-L3
The value of `name` is an English word and is the same as the folder name.
The value of `title` is the whole name of the technical group.
Here is the reference: https://github.com/TGmeetup/TGmeetup/blob/master/CONTRIBUTING.md#add-a-community
Many thanks.